scripts/generate_bindings.sh

```diff
@@ -42,7 +42,7 @@ bindgen $common_params \
 bindgen $common_params \
   -- \
   -I"$PLAYDATE_C_API" \
-  -I"$(which arm-none-eabi-gcc)/../arm-none-eabi/include" \
+  -I"$($(which arm-none-eabi-gcc) -print-sysroot)/include" \
```

Comment:
```suggestion
  -I"$(arm-none-eabi-gcc -print-sysroot)/include" \
```

Can you still take out the `which` part too? It shouldn't be necessary

Thank you for this change!

Reply:
Oh, hah, I got caught up in the rest and missed that bit.  Thanks!